apps/dashboard/src/constants/chains.ts

```diff
@@ -54,3 +55,17 @@ export const defaultChains: Chain[] = [
   avalancheFuji,
   localhost,
 ];
+
+export type SUPPORTED_CHAIN_ID =
```

Comment:
conceptually we'll want to remove _all_ of this - what is this used for? Can we get rid of it? If not that's fine but this is a leftover thing that v5 has no concept of

Reply:
It's being used in 2 places. 1 is a component that I'm unfamiliar with, the other one is for fetching token balances from Moralis - I assume the list of chain ids is also the list of supported chainIds from Moralis.

We can simply change this type to just `number` if you want ^

References:
1. src/lib/deployment/template-values.ts
2. src/pages/api/moralis/balances.ts

Reply:
the goal would be to change all the places this is used to allow all chainids as type that chainlist has, meaning yeah likely a `number` and then we need to check at runtime if the number that is passed is a valid chain id (if it matters in that component)